src/main/webapp/app/exercises/shared/team-config-form-group/team-config-form-group.component.ts

```diff
@@ -1,21 +1,29 @@
-import { Component, Input, OnInit } from '@angular/core';
+import { AfterViewChecked, Component, Input, OnDestroy, OnInit, ViewChild } from '@angular/core';
 import { TeamAssignmentConfig } from 'app/entities/team-assignment-config.model';
 import { cloneDeep } from 'lodash-es';
 import { Exercise, ExerciseMode } from 'app/entities/exercise.model';
 import { ModePickerOption } from 'app/exercises/shared/mode-picker/mode-picker.component';
+import { NgModel } from '@angular/forms';
+import { Subject, Subscription } from 'rxjs';
 
 @Component({
     selector: 'jhi-team-config-form-group',
     templateUrl: './team-config-form-group.component.html',
     styleUrls: ['./team-config-form-group.component.scss'],
 })
-export class TeamConfigFormGroupComponent implements OnInit {
+export class TeamConfigFormGroupComponent implements AfterViewChecked, OnDestroy, OnInit {
     readonly INDIVIDUAL = ExerciseMode.INDIVIDUAL;
     readonly TEAM = ExerciseMode.TEAM;
 
     @Input() exercise: Exercise;
     @Input() isImport: boolean;
 
+    @ViewChild('minTeamSize') minTeamSizeField?: NgModel;
```

Comment:
Same point here - the check could probably be added to the (existing) ngModelChange